spec/vmpooler/api_spec.rb

```diff
@@ -11,13 +11,20 @@ def app()
   describe 'Dashboard' do
 
     context '/' do
+      before { get '/' }
+
+      it { expect(last_response.status).to eq(302) }
+      it { expect(last_response.location).to match(/dashboard/) }
```

Comment:
This will regex match "dashboard" anywhere in the location. Do you want this or `eq('/dashboard/')` to verify the location is exactly just "/dashboard/"?